tests/integration/utilities.py

```diff
@@ -0,0 +1,224 @@
+import psycopg2
+import requests
+
+
+def populate_firestore():
```

Comment:
It might make the tests easier to understand and maintain if we set up just the data required for each test in the test, rather than setting up everything in one place for every test